lib/files.go

```diff
@@ -0,0 +1,194 @@
+package lib
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+// AppendIfMissing appends the content string to the given file
+func AppendIfMissing(filePath string, content string) error {
+	fileToModify, err := os.OpenFile(filePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	Check(err)
+
+	fileContents, err := ioutil.ReadFile(fileToModify.Name())
+	Check(err)
+
+	if !strings.Contains(string(fileContents), content) {
+		_, err = fileToModify.Write([]byte(content))
```

Comment:
I think the old implementation with `echo` implicitly added a newline, but this won't necessarily add a newline unless I've misread